tools/perf_k8svcs_probe.yaml

```diff
@@ -0,0 +1,100 @@
+# 2 services will get istio injected
```

Comment:
why a new file, can you edit the current file as I can't tell the changes anymore
(and it would create too much noise, also you checked in the .gitignore generated yaml)

Reply:
I don't have a plan to merge this PR. 😀

Reply:
well I think you should once you get it working, adding readyness/liveness to the perf setup is a good idea